app/helpers/dropdown/invoice_sending.rb

```diff
@@ -30,7 +30,8 @@ def send_links
     end
 
     def add_item(key, options = {})
-      path = template.group_invoice_list_path(template.parent, options)
+      group = template.parent.is_a?(Group) ? template.parent : template.parent.group
```

Comment:
Auch hier machts eigentlich nur einen Unterschied, ob es eine Invoice oder eine InvoiceList ist, oder liege ich da falsch?
```suggestion
      group = template.is_a?(InvoiceList) ? template.parent : template.parent.group
```
(ungetestet)